lib/railway_ui_web/templates/consumed_message/index.html.eex

```diff
@@ -1,29 +1,28 @@
 <div class="row justify-content-md-center">
   <div class="col-md-auto">
-    <h1 class="display-3">Consumed Messages
-    </h1>
-    <table class="table table-responsive pt-3 table-hover">
-      <thead>
+    <h1 class="display-3">Consumed Messages</h1>
+    <table class="table table-bordered table-responsive table-hover">
+      <thead class="thead-dark">
         <tr>
-          <th>Message Type</th>
-          <th>UUID</th>
-          <th>Correlation ID</th>
-          <th>User UUID</th>
-          <th>Queue</th>
-          <th>Exchange</th>
-          <th>Encoded Message</th>
+          <th scope="col">Encoded Message</th>
```

Comment:
Moved this up in the order of the table as I find it is the #1 thing I care about.